src/containers/App.js

```diff
@@ -2,24 +2,39 @@ import React, { Component } from 'react';
 import { loadMovies } from '../actions';
 import { fetchData } from '../utilz/apiCalls';
 import { connect } from 'react-redux';
+import MoviesDisplay from './MoviesDisplay';
 // import '../App.css';
 
 class App extends Component {
+constructor(props) {
+  super()
+
+}
 componentDidMount = async () => {
   const { handleFetch } = this.props;
   fetchData()
-  .then(data => handleFetch(data.results))
+  .then(data => data)
+  .then(movies => handleFetch(movies))
 }
 
   render() {
     return (
-      <h1>Yo World</h1>
+      <main>
+        {
+          (this.props.movies.length) && 
```

Comment:
Curious about what this line is doing exactly

Reply:
This line is confirming that movie data exists in our state.  When this.props.movies.length evaluates to truthy, it will move past the and operator and render the movie container, triggering the map that creates all of the MoviePoster components.   =]